launch/resolve-ip.yml

```diff
@@ -11,7 +10,7 @@ expose:
   health_check:
     type: http
     path: /healthcheck
-elbs:
-  production:
-  - resolve-ip
+- name: pprof
+  port: 6060
+  bind: 6060
```

Comment:
is removing the `elbs` section here intentional (if so, what happens with this change)?